openedx_events/learning/data.py

```diff
@@ -190,3 +190,29 @@ class CourseDiscussionConfigurationData:
     unit_level_visibility = attr.ib(type=bool, default=False)
     plugin_configuration = attr.ib(type=dict, default={})
     contexts = attr.ib(type=List[DiscussionTopicContext], factory=list)
+
+
+@attr.s(frozen=True)
+class GradeData:
```

Comment:
As this event is associated with the class [PersistentCourseGrade](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/grades/models.py#L537)  I think it would be better to change it to **PersistentCourseGradeData** to avoid confusion

Reply:
Agreed. Please modify the description and naming if this data class can't be used for all grade-related events.